CMakeLists.txt

```diff
@@ -255,9 +205,14 @@ target_compile_definitions(remill_settings INTERFACE
   "REMILL_BUILD_SEMANTICS_DIR_SPARC64=\"${REMILL_BUILD_SEMANTICS_DIR_SPARC64}\""
 )
 
-set(THIRDPARTY_LIBRARY_LIST thirdparty_llvm thirdparty_xed thirdparty_glog thirdparty_gflags thirdparty_sleigh)
 target_link_libraries(remill_settings INTERFACE
```

Comment:
should deps go into _settings as well?

Reply:
It depends on how precise you want to be with the linking of the dependencies to the various libraries/executables.

Ideally, you only link to the dependencies you need, but adding them here means you only need to change one spot 🤷